### DISCUSSION TITLE
Lower the entry barrier to start with the automator

### DISCUSSION THREAD

**Discussion by lechnerc77** [SAP-archive/btp-setup-automator#120] [Ideas]
Hi friends of the BTP Setup Automator,

I just had a discussion on how to lower the entry barrier for the tool beyond the existing documentation. One idea would be a short video on how to get started with the tool (the very first steps) that we link on the page.

What do you think about that? Other ideas are of course also welcome

Cheers
Christian

**Comment by lechnerc77** [comment]
okay, so here we go: a short video that should give you a kick start: 
[#QuickGlance - the btp-setup-automator](https://youtu.be/BHBgQ45fgIk)

**Reply by rui1610** [reply]
Fantastic Christian. Putting the video as well on the README.md file helps users to get started quicker.
Thanks a lot.

**Comment by yevgentrukhin** [comment]
Awesome, thanks!
